pages/common/pandoc.md

```diff
@@ -1,7 +1,15 @@
 # pandoc
 
-> General markup converter.
+> Convert documents between various formats.
 
 - Convert file to pdf (the output format is automatically determined from the output file's extension):
 
 `pandoc {{input.md}} -o {{output.pdf}}`
+
+- Convert a file to a specific output format (useful for when the extension alone is ambiguous):
+
+`pandoc {{input.docx}} -t {{markdown_github}} -o {{output.md}}`
```

Comment:
I don't see markdown_github being one of the output formats here - https://linux.die.net/man/1/pandoc. Is this outdated ?